cost/budget_alerts/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v1.13
+
+- Added the filteration logic for getting only the Parent BC details
```

Comment:
I thought the goal was to get the parent BC for Unallocated.  Also the README should probably be updated to include any changes in the filtering it's doing.

Reply:
Also you if you haven't already you should compare the old report with your new report and make sure it matches your expectations.  Also compare to Optima.

Reply:
What about other Billing Centers that are not Unallocated.  What is the new behavior going to do with them?

Reply:
Hi @cdwilhelm , The issue may appear if  suppose there is a parent BC with name `Test` and multiple child BC with similar name, So i have included the filter condition to get value of Parent BC only. Please let me know if i should make the change only for `Unallocated` only. I have compared the result using an older policy which was in the Rightscale portal and the latest one, and i have checked the same in the Optima also. PFB snapshot of the same:
This is with the older policy:
![image](https://user-images.githubusercontent.com/61958294/88875568-1a0c2e80-d23f-11ea-9244-893d3c27eec5.png)

This is of the modified policy:
![image](https://user-images.githubusercontent.com/61958294/88875772-838c3d00-d23f-11ea-8b33-321b171b42ac.png)

and this is the value from optima:
![image](https://user-images.githubusercontent.com/61958294/88875986-e7166a80-d23f-11ea-9144-417154488701.png)
Thanks

Reply:
For other BC also, it will fetch the values from Parent BC only, and there should not be any data mismatch, like the unallocated BC. PFB snap of the values of `Test` BC :
from older policy:
![image](https://user-images.githubusercontent.com/61958294/88876333-ae2ac580-d240-11ea-8324-c73b1f552422.png)

from the modified policy:
![image](https://user-images.githubusercontent.com/61958294/88876373-c7337680-d240-11ea-81be-00c32aedc56a.png)

in the optima:
![image](https://user-images.githubusercontent.com/61958294/88876442-e6ca9f00-d240-11ea-8472-4889b95edfe9.png)
Thanks

Reply:
Here in the above case, since there was no child BC with name `Test`, there is similar data in both the older and the newer policy. Thanks. Should i change the filter condition for BC `Unallocated` only? If there are other multiple BC with similar name, in that case data mismatch can be there. Thanks
cc @chandravurubindi